tests/test_columns.py

```diff
@@ -33,6 +33,8 @@ class Product(orm.Model):
         "created": orm.DateTime(default=datetime.datetime.now),
         "created_day": orm.Date(default=datetime.date.today),
         "created_time": orm.Time(default=time),
+        "created_auto_now_add": orm.Date(auto_now_add=True),
```

Comment:
Still missing coverage here. You need a Date with auto_now=True.

I think it would make sense to have all 4 combinations of Date, DateTime with args auto_add and auto_add_now ?